src/atoms/Auth.ts

```
@@ -1,13 +1,79 @@
+export const ANONYMOUS_USER_TEMPLATE: User = {
+  id: '0',
+  nickname: '비회원',
+  mobilityTools: [],
+  isNewsLetterSubscriptionAgreed: false,
+};
+
+export const isAnonymousUserAtom = atom(get => {
+  const userInfo = get(userInfoAtom);
+  // 비회원 체크: nickname이 '비회원'인 경우
+  // (id가 '0'인 경우는 레거시 케이스이며, 실제로는 채번된 userId를 가짐)
+  return userInfo?.id === '0' || userInfo?.nickname === ANONYMOUS_USER_TEMPLATE.nickname;
```

ESLint

[error] Replace `userInfo?.id·===·'0'·||·userInfo?.nickname·===·ANONYMOUS_USER_TEMPLATE.nickname` with `(⏎····userInfo?.id·===·'0'·||⏎····userInfo?.nickname·===·ANONYMOUS_USER_TEMPLATE.nickname⏎··)`

(prettier/prettier)